api/pkg/apis/projectcalico/v3/felixconfig.go

```diff
@@ -195,7 +195,7 @@ type FelixConfigurationSpec struct {
 
 	// MetadataAddr is the IP address or domain name of the server that can answer VM queries for
 	// cloud-init metadata. In OpenStack, this corresponds to the machine running nova-api (or in
-	// Ubuntu, nova-api-metadata). A value of none (case insensitive) means that Felix should not
+	// Ubuntu, nova-api-metadata). A value of none (case-insensitive) means that Felix should not
```

Comment:
hyphenation rules are complicated. It's quite likely I'll make mistakes (I'm human, really!)